challenge_5/java/jdfurlan/src/FindTheDifference.java

```diff
@@ -0,0 +1,21 @@
+
+public class FindTheDifference {
+
+	public static void main(String[] args) {
+		String s = "abcd";
+		String t = "dbeca";
+
+		boolean[] b = new boolean[128];
```

Comment:
It does work with multiple of the same char, try apple and lepaip, it will show 'i' look closer at the logic.